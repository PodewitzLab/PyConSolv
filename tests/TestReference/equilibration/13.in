minimization
&cntrl
	imin=1,ncyc=500,maxcyc=1000,ntr=1,cut=8.0,
	restraint_wt=10.0,restraintmask="!@H=&:1",
/
