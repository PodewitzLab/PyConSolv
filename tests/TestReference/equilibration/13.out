
          -------------------------------------------------------
          Amber 20 PMEMD                              2020
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 18

|  Compiled date/time: Sat Aug 20 11:06:42 2022
| Run on 09/01/2023 at 17:31:48

|   Executable path: pmemd.cuda
| Working directory: /home/rat/PYCONSOLV_DATA/v1/equilibration
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|   MDIN: 13.in                                                                 
|  MDOUT: 13.out                                                                
| INPCRD: 12.rst7                                                               
|   PARM: LIG_solv.prmtop                                                       
| RESTRT: 13.rst7                                                               
|   REFC: 12.rst7                                                               
|  MDVEL: mdvel                                                                 
|   MDEN: mden                                                                  
|  MDCRD: mdcrd                                                                 
| MDINFO: mdinfo                                                                
|  MDFRC: mdfrc                                                                 


 Here is the input file:

minimization                                                                   
&cntrl                                                                         
	imin=1,ncyc=500,maxcyc=1000,ntr=1,cut=8.0,                                    
	restraint_wt=10.0,restraintmask="!@H=&:1",                                    
/                                                                              


Note: ig = -1. Setting random seed to   862875 based on wallclock time in 
      microseconds.
| irandom = 1, using AMBER's internal random number generator (default).
 
|--------------------- INFORMATION ----------------------
| GPU (CUDA) Version of PMEMD in use: NVIDIA GPU IN USE.
|                    Version 18.0.0
| 
|                      03/25/2018
| 
| Implementation by:
|                    Ross C. Walker     (SDSC)
|                    Scott Le Grand     (nVIDIA)
| 
| Version 18 performance extensions by:
|                    David Cerutti     (Rutgers)
| 
| Precision model in use:
|      [SPFP] - Single Precision Forces, 64-bit Fixed Point
|               Accumulation. (Default)
| 
|--------------------------------------------------------
 
|----------------- CITATION INFORMATION -----------------
|
|    When publishing work that utilized the CUDA version
|    of AMBER, please cite the following in addition to
|    the regular AMBER citations:
|
|  - Romelia Salomon-Ferrer; Andreas W. Goetz; Duncan
|    Poole; Scott Le Grand; Ross C. Walker "Routine
|    microsecond molecular dynamics simulations with
|    AMBER - Part II: Particle Mesh Ewald", J. Chem.
|    Theory Comput., 2013, 9 (9), pp3878-3888,
|    DOI: 10.1021/ct400314y.
|
|  - Andreas W. Goetz; Mark J. Williamson; Dong Xu;
|    Duncan Poole; Scott Le Grand; Ross C. Walker
|    "Routine microsecond molecular dynamics simulations
|    with AMBER - Part I: Generalized Born", J. Chem.
|    Theory Comput., 2012, 8 (5), pp1542-1555.
|
|  - Scott Le Grand; Andreas W. Goetz; Ross C. Walker
|    "SPFP: Speed without compromise - a mixed precision
|    model for GPU accelerated molecular dynamics
|    simulations.", Comp. Phys. Comm., 2013, 184
|    pp374-380, DOI: 10.1016/j.cpc.2012.09.022
|
|--------------------------------------------------------
 
|------------------- GPU DEVICE INFO --------------------
|
|            CUDA_VISIBLE_DEVICES: not set
|   CUDA Capable Devices Detected:      1
|           CUDA Device ID in use:      0
|                CUDA Device Name: NVIDIA GeForce RTX 3070 Laptop GPU
|     CUDA Device Global Mem Size:   8191 MB
| CUDA Device Num Multiprocessors:     40
|           CUDA Device Core Freq:   1.56 GHz
|
|--------------------------------------------------------
 
 
| Conditional Compilation Defines Used:
| PUBFFT
| BINTRAJ
| CUDA
| EMIL

| Largest sphere to fit in unit cell has radius =    17.630

| New format PARM file being parsed.
| Version =    1.000 Date = 09/01/23 Time = 17:28:38

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 getting box info from netcdf restart file
 NATOM  =    6251 NTYPES =       7 NBONH =    4686 MBONA  =    1564
 NTHETH =      12 MTHETA =       6 NPHIH =      18 MPHIA  =       0
 NHPARM =       0 NPARM  =       0 NNB   =   10967 NRES   =    1565
 NBONA  =    1564 NTHETA =       6 NPHIA =       0 NUMBND =       9
 NUMANG =      10 NPTRA  =       1 NATYP =       9 NPHB   =       0
 IFBOX  =       1 NMXRS  =       4 IFCAP =       0 NEXTRA =    1560
 NCOPY  =       0

| Coordinate Index Table dimensions:     8    8    7
| Direct force subcell size =     4.5656    4.5511    5.0372

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =      50, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  10.00000

Energy minimization:
     maxcyc  =    1000, ncyc    =     500, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =       1

Extra-points options:
     frameon =       1, chngmask=       1

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   36.525   Box Y =   36.408   Box Z =   35.260
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   40       NFFT2 =   40       NFFT3 =   36
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  default_name                                                                    
     Mask !@H=&:1; matches     1 atoms
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  4686     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =  4686     0
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  1564     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =     4     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =    12     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =    12     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =     6     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =     6     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =    18     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =    18     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =     0     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =     0     0

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 
 Number of triangulated 3-point waters found:     1560

     Sum of charges from parm topology file =   0.00000756
     Forcing neutrality...

| Dynamic Memory, Types Used:
| Reals              294880
| Integers           295091

| Nonbonded Pairs Initial Allocation:     1044073

| GPU memory information (estimate):
| KB of GPU memory in use:     23067
| KB of CPU memory in use:     10467

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -2.8130E+04     3.2301E-02     3.8047E+00     PT          1

 BOND    =     5210.2617  ANGLE   =        0.7971  DIHED      =        0.0000
 VDWAALS =     5690.0107  EEL     =   -38818.6583  HBOND      =        0.0000
 1-4 VDW =       -0.1455  1-4 EEL =     -212.1707  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     50      -2.8130E+04     1.2487E-02     5.3263E-01     CL9        11

 BOND    =     5210.2938  ANGLE   =        0.7996  DIHED      =        0.0000
 VDWAALS =     5689.9994  EEL     =   -38818.8442  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0271  RESTRAINT  =        0.0015
 EAMBER  =   -28129.9235


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100      -2.8130E+04     1.2476E-02     5.3110E-01     CL9        11

 BOND    =     5210.2940  ANGLE   =        0.7996  DIHED      =        0.0000
 VDWAALS =     5689.9994  EEL     =   -38818.8595  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0266  RESTRAINT  =        0.0015
 EAMBER  =   -28129.9382


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    150      -2.8130E+04     1.2464E-02     5.3011E-01     CL9        11

 BOND    =     5210.2943  ANGLE   =        0.7996  DIHED      =        0.0000
 VDWAALS =     5689.9992  EEL     =   -38818.8509  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0261  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9289


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -2.8130E+04     1.2453E-02     5.2853E-01     CL9        11

 BOND    =     5210.2945  ANGLE   =        0.7996  DIHED      =        0.0000
 VDWAALS =     5689.9994  EEL     =   -38818.8548  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0257  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9319


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    250      -2.8130E+04     1.2437E-02     5.2645E-01     CL9        11

 BOND    =     5210.2947  ANGLE   =        0.7997  DIHED      =        0.0000
 VDWAALS =     5689.9994  EEL     =   -38818.8570  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0251  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9334


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300      -2.8130E+04     1.2427E-02     5.2512E-01     CL9        11

 BOND    =     5210.2949  ANGLE   =        0.7997  DIHED      =        0.0000
 VDWAALS =     5689.9994  EEL     =   -38818.8535  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0247  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9293


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    350      -2.8130E+04     1.2418E-02     5.2388E-01     CL9        11

 BOND    =     5210.2951  ANGLE   =        0.7997  DIHED      =        0.0000
 VDWAALS =     5689.9995  EEL     =   -38818.8471  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0244  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9224


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400      -2.8130E+04     1.2408E-02     5.2245E-01     CL9        11

 BOND    =     5210.2952  ANGLE   =        0.7997  DIHED      =        0.0000
 VDWAALS =     5689.9992  EEL     =   -38818.8779  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0240  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9529


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    450      -2.8130E+04     1.2397E-02     5.2107E-01     CL9        11

 BOND    =     5210.2954  ANGLE   =        0.7997  DIHED      =        0.0000
 VDWAALS =     5689.9991  EEL     =   -38818.8553  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0236  RESTRAINT  =        0.0016
 EAMBER  =   -28129.9298


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -2.8130E+04     1.2387E-02     5.1977E-01     CL9        11

 BOND    =     5210.2956  ANGLE   =        0.7997  DIHED      =        0.0000
 VDWAALS =     5689.9992  EEL     =   -38818.8581  HBOND      =        0.0000
 1-4 VDW =       -0.1450  1-4 EEL =     -212.0233  RESTRAINT  =        0.0017
 EAMBER  =   -28129.9318


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    550      -2.8130E+04     3.7403E-02     1.0597E+00     H2       4690

 BOND    =     5210.6225  ANGLE   =        0.7550  DIHED      =        0.0000
 VDWAALS =     5690.5191  EEL     =   -38819.9888  HBOND      =        0.0000
 1-4 VDW =       -0.1457  1-4 EEL =     -212.2254  RESTRAINT  =        0.0878
 EAMBER  =   -28130.4633

     .... RESTARTED DUE TO LINMIN FAILURE ...

     .... RESTARTED DUE TO LINMIN FAILURE ...


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600      -2.8130E+04     1.3893E-02     5.3453E-01     H7          8

 BOND    =     5211.2777  ANGLE   =        0.8692  DIHED      =        0.0000
 VDWAALS =     5691.3650  EEL     =   -38821.9709  HBOND      =        0.0000
 1-4 VDW =       -0.1449  1-4 EEL =     -211.9535  RESTRAINT  =        0.0708
 EAMBER  =   -28130.5574

     .... RESTARTED DUE TO LINMIN FAILURE ...


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    650      -2.8131E+04     3.2502E-02     8.1564E-01     H1       4689

 BOND    =     5211.1606  ANGLE   =        0.8235  DIHED      =        0.0000
 VDWAALS =     5691.4514  EEL     =   -38821.9004  HBOND      =        0.0000
 1-4 VDW =       -0.1449  1-4 EEL =     -211.9658  RESTRAINT  =        0.0697
 EAMBER  =   -28130.5756

     .... RESTARTED DUE TO LINMIN FAILURE ...

     .... RESTARTED DUE TO LINMIN FAILURE ...


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    697      -2.8131E+04     1.9277E-02     4.5090E-01     H1       4689

 BOND    =     5211.1689  ANGLE   =        0.8234  DIHED      =        0.0000
 VDWAALS =     5691.4514  EEL     =   -38821.9061  HBOND      =        0.0000
 1-4 VDW =       -0.1449  1-4 EEL =     -211.9666  RESTRAINT  =        0.0697
 EAMBER  =   -28130.5740

     ***** REPEATED LINMIN FAILURE *****
--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|  NonSetup CPU Time in Major Routines:
|
|     Routine           Sec        %
|     ------------------------------
|     Nonbond           0.33   62.27
|     Bond              0.00    0.00
|     Angle             0.00    0.00
|     Dihedral          0.00    0.00
|     Shake             0.00    0.00
|     Other             0.20   37.73
|     ------------------------------
|     Total             0.53

|  PME Nonbond Pairlist CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     Set Up Cit           0.00    0.00
|     Build List           0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  PME Direct Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     NonBonded Calc       0.00    0.00
|     Exclude Masked       0.00    0.00
|     Other                0.00    0.11
|     ---------------------------------
|     Total                0.00    0.11

|  PME Reciprocal Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     1D bspline           0.00    0.00
|     Grid Charges         0.00    0.00
|     Scalar Sum           0.00    0.00
|     Gradient Sum         0.00    0.00
|     FFT                  0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  Setup CPU time:            1.01 seconds
|  NonSetup CPU time:         0.53 seconds
|  Total CPU time:            1.54 seconds     0.00 hours

|  Setup wall time:           0    seconds
|  NonSetup wall time:        1    seconds
|  Total wall time:           1    seconds     0.00 hours
